src/main/flight/pid.c

```diff
@@ -615,10 +616,12 @@ void pidInitConfig(const pidProfile_t *pidProfile)
 #if defined(USE_SMART_FEEDFORWARD)
     smartFeedforward = pidProfile->smart_feedforward;
 #endif
+
 #if defined(USE_ITERM_RELAX)
     itermRelax = pidProfile->iterm_relax;
     itermRelaxType = pidProfile->iterm_relax_type;
     itermRelaxCutoff = pidProfile->iterm_relax_cutoff;
+    itermRelaxSetpointThreshold = ITERM_RELAX_SETPOINT_THRESHOLD * 20.0f / itermRelaxCutoff;
```

Comment:
Looking at how the code now is, I don't think `pidProfile->iterm_relax_cutoff` needs to be copied into DTCM RAM any more - it is not used anywhere inside the PID loop.

Reply:
Not sure if I've addressed these in the ideal way... hopefully OK.
Still no clue about the failed unit test.  How do I get the details of what failed and why?

Reply:
Looks good now.
I think you should really set yourself up to run the tests in your build environment - doing this kind of change and not being able to verify locally that the tests are not affected introduces way to much delay for every iteration you need to take to fix the tests.
What is your build environment?

Reply:
eTracer has kindly tried to get a local test environment working... I have MacOSX Mohave with everything up to date, but get errors when performing the unit test that we cannot resolve.

Is it possible for the Travis tests done here on GitHub to report the actual errors?  They just say 'failed Error2' but no actual details of the error like which line and what values.  That hamstrings people like me.

Reply:
Where does the 20 stand for?

Reply:
20 is the default value for itermRelaxCutoff.  
I am using this calculation to compensate for the change in amplitude of the itermRelaxFactor associated with changes in cutoff.  
Since the default is 20, this line reduces to itermRelaxSetpointThreshold = ITERM_RELAX_SETPOINT_THRESHOLD, and all the subsequent maths is the same.

Reply:
You should name this default so everyone can know what it is.

Reply:
Ah OK good point